frontend/src/views/CreditsPage/CreditsPage.jsx

```diff
@@ -0,0 +1,47 @@
+import './CreditsPage.scss';
+
+import { Typography } from '@mui/material';
+import Button from '@mui/material/Button';
+
+import buttonStyles from '../../components/Button/Button.module.scss';
+
+const ButtonsComponent = () => {
```

Comment:
This name may be improved. It is only used in the CreditsPage return function so it must be a Component.

The better name may be DeveloperButtons.